docs/writing_to_promscale.md

```diff
@@ -0,0 +1,173 @@
+http://localhost:9201/write
+
+
+This endpoint only supports the HTTP POST method.
+
+
+The default format for this endpoint are protocol buffers but Promscale also supports a JSON streaming format which is detailed in the next sections.
+
+
+If you are using Prometheus, simply configure `remote_write` to point to this endpoint and you are done. If, however, you are writing a custom application to push data to Promscale, keep reading.
+
+
+## General time-series format rules
+A time-series must always contain both labels and samples.
+
+
+Labels are a collection of label name/value pairs with unique label names. Every request must  contain a label named `__name__` which cannot have an empty value. All other labels are optional metadata.
+
+
+A series entry always contains two things:
+* An integer timestamp in milliseconds since epoch, i.e. 1970-01-01 00:00:00 UTC, excluding leap second, represented as required by Go's [ParseInt](https://golang.org/pkg/strconv/#ParseInt) function. 
+* Floating point number that represents the actual measured value.
+
+## JSON streaming format
+
+This format was introduced in Promscale to enable easier usage of the endpoint when ingesting metric data from 3rd party tools. It is not part of the `remote_write` specification for Prometheus. It is slightly less efficient to use this format than the Protobuf format. 
+
+
+This format is a JSON stream of objects with two fields:
+* `labels` JSON object in which all the labels are represented as fields
+* `samples` JSON array which contains arrays of timestamp-value sets
+
+Here is a short example of what a simple payload would look like:
+
+```
+{
+    "labels":{"__name__": "cpu_usage", "namespace":"dev", "node": "brain"},
+    "samples":[
+        [1577836800000, 100],
+        [1577836801000, 99],
+        [1577836802000, 98],
+        ...
+    ]
+}
+{
+    "labels":{"__name__": "cpu_usage", "namespace":"prod", "node": "brain"},
+    "samples":[
+        [1577836800000, 98],
+        [1577836801000, 99],
+        [1577836802000, 100],
+        ...
+    ]
+}
+...
+```
+
+As you can see, the labels object is pretty self explanatory. Fields represent the label name and value is the label value. Samples array consists of arrays with some strict rules:
+* There can be only two values in the array
+* First value is a UNIX timestamp in milliseconds which has to be an integer value
+* Second value is a floating point number which represents the corresponding value of the metric.
+
+
+In order to send a request to Promscale, you would need to send an HTTP POST request with the request body set to the JSON payload and set the required header values:
+* `Content-Type` header should be set to `application/json`
+* `Content-Encoding` header can optionally be set to `snappy` if you are using snappy compression on the JSON payload
```

Comment:
I think this header is optional, but the way this sounds is, **it can be optionally set to snappy** (else set to what?).

How about:
```
* (OPTIONAL, if using snappy)`Content-Encoding` header should be set to `snappy` if you are using snappy compression on the JSON payload
```

Reply:
I'd use "If using Snappy compression set `Content-Encoding` header to `snappy`, otherwise leave unset."